apps/engineeringblog/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "engineeringblog",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
```

Comment:
We'll need also to have a script for cleaning  whatever we have installed

` "clean": "rm -rf .next .turbo node_modules",
`